udata_front/theme/gouvfr/datagouv-components/src/components/QualityComponentInline/QualityComponentInline.vue

```diff
@@ -1,78 +1,77 @@
-                    class="fr-my-1w"
-                />
-                <QualityItem
-                    :passed="quality.resources_documentation"
-                    :messagePassed='$t("Files documented")'
-                    :messageFailed='$t("Files documentation missing")'
-                    class="fr-my-1w"
-                />
-                <QualityItem
-                    :passed="quality.license"
-                    :messagePassed='$t("License filled")'
-                    :messageFailed='$t("No license set")'
-                    class="fr-my-1w"
-                />
-                <QualityItem
-                    :passed="quality.update_frequency && !!quality.update_fulfilled_in_time"
-                    :messagePassed='$t("Update frequency followed")'
-                    :messageFailed='quality.update_frequency ? $t("Update frequency not followed") : $t("Update frequency not set")'
-                    class="fr-my-1w"
-                />
-                <QualityItem
-                    :passed="quality.has_open_format"
-                    :messagePassed='$t("File formats are open")'
-                    :messageFailed='$t("File formats are closed")'
-                    class="fr-my-1w"
-                />
-                <QualityItem
-                    :passed="quality.temporal_coverage"
-                    :messagePassed='$t("Temporal coverage filled")'
-                    :messageFailed='$t("Temporal coverage not set")'
-                    class="fr-my-1w"
-                />
-                <QualityItem
-                    :passed="quality.spatial"
-                    :messagePassed='$t("Spatial coverage filled")'
-                    :messageFailed='$t("Spatial coverage not set")'
-                    class="fr-my-1w"
-                />
-                <QualityItem
-                    :passed="quality.all_resources_available"
-                    :messagePassed='$t("All files are available")'
-                    :messageFailed='$t("Some files are unavailable")'
-                    class="fr-my-1w"
-                />
-                <div class="fr-grid-row fr-grid-row--right not-enlarged">
-                    <a
-                    :href="config.guides_quality_url"
-                    target="_blank"
-                    rel="noopener"
-                    :title="$t('Learn more about this indicator - opens a new window')"
-                    >
-                    {{$t("Learn more about this indicator")}}
-                    </a>
-                </div>
-            </template>
-        </Toggletip>
-        <p class="fr-m-0 fr-mr-1v">
-            {{$t('Metadata quality:')}}
-        </p>
-        <QualityScore
-            :score="quality.score"
-        />
+        <div class="fr-grid-row fr-grid-row--middle fr-ml-n3v">
+          <Toggletip
+              class="fr-btn fr-btn--tertiary-no-outline fr-btn--secondary-grey-380 fr-icon-information-line fr-icon--sm"
```

Comment:
I think we'll have to make a new class for this really small button. Using negative margin around is not working on hover state.